addon/components/infinity-loader.js

```diff
@@ -1,4 +1,11 @@
 import Ember from 'ember';
+import emberVersionIs from 'ember-version-is';
+
+if (emberVersionIs('lessThan', '1.13.0')) {
+  Ember.Component.reopen({
```

Comment:
This is going to effect all components in the host application.  Could we do it in the extend call below somehow instead?  I don't want to effect the rest of a developer's application.  Worse comes to worse - we could do the reopen before the export just on the `infinity-loader`

Thanks @davidgoli !